.githooks/pre-commit

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+VALID_BRANCH_REGEX="^(master$|dev\/.+)"
+
+if [[ ! $(git rev-parse --abbrev-ref HEAD) =~ $VALID_BRANCH_REGEX ]]
```

Comment:
This will cause problems in detached head state :( Thus, it makes using
```
$ git commit --amend
```
during interactive rebase impossible.

Reply:
We should probably use a pre-push hook instead.

Reply:
This should do the trick
```
$ mv .githooks/pre-commit .githooks/pre-push
```
Works fine for me locally.